internal/operands/node-labeller/defaults.go

```diff
@@ -0,0 +1,8 @@
+package node_labeller
+
+const (
+	kubevirtNodeLabellerDefaultImage = "quay.io/kubevirt/node-labeller:v0.2.0"
+	kvmInfoNfdDefaultImage           = "quay.io/kubevirt/kvm-info-nfd-plugin:v0.5.8"
+	kvmCpuNfdDefaultImage            = "quay.io/kubevirt/cpu-nfd-plugin:v0.1.1"
+	libvirtDefaultImage              = "docker.io/kubevirt/virt-launcher:v0.21.0"
```

Comment:
I think we can update this image to version `v0.35.0`

Reply:
Ok, I will open another PR for it.